### PR TITLE
Add Instructions To Install 100% In Android

### PR DESCRIPTION
**Pull request by alixgamer175**
Thx

Fixes #

### CODE REVIEW

**Review by alixgamer175 — COMMENTED** [review]
This Will Install 100% Code Server In Your Android Phone

File: docs/android.md
Comment (jsjoeio) [inline docs/android.md:6]: Can we change this to a numbered list?

File: docs/android.md
Comment (jsjoeio) [inline docs/android.md:11]: And this?

File: docs/android.md
Comment (jsjoeio) [inline docs/android.md:16]: Should this have `-g`?

**Review by jsjoeio — CHANGES_REQUESTED** [review]
Thank you so much for adding this! 🎉

I guess this wouldn't fall under the Termux docs since it uses userland. I have some minor requests about formatting but generally, this looks great!

(note to self: I'll probably need to refactor and combine some stuff later)

File: docs/android.md
Comment (alixgamer175) [reply docs/android.md:6]: Ofc

File: docs/android.md
Comment (alixgamer175) [reply docs/android.md:16]: No that's crazy

**Comment by alixgamer175** [comment]
If you want to add some other docs then i can help
I doesn't know much languages
I used every type of linux in android

File: docs/android.md
Comment (im-coder-lg) [reply docs/android.md:6]: Oh, I hope you don't mind me looping in... `sudo` actually doesn't exist on Termux, and I think it's kinda hard to install it since the `sudo` package(aka `tsu`) is kinda a problem. I can't try installing it without a mirror error. So, maybe we need to remove `sudo` in line 5.

Comment (im-coder-lg) [reply docs/android.md:6]: Sorry for that...

Comment (jsjoeio) [reply docs/android.md:6]: Awesome! Let me know after you fix it and then we can get this merged!

**Review by jsjoeio — CHANGES_REQUESTED** [review]

**Comment by jsjoeio** [comment]
> If you want to add some other docs then i can help

Thank you for offering that! Let's get this updated and merged and then discuss other doc improvements ♥️

**Comment by jsjoeio** [comment]
Still waiting on those changes before we can review and merge this! Going to push to the next release.

File: docs/android.md
Comment (alixgamer175) [reply docs/android.md:6]: It’s not termux
It’s userland on playstore

**Comment by alixgamer175** [comment]
ABA3FECD-1F8A-42C9-AE99-B9DC0382D761
I didn't use fmt with yarn !!

**Comment by jsjoeio** [comment]
@alixgamer175 do you know if you hit "Allow edit from maintainers"? If so, I can fix this for you

**Comment by jsjoeio** [comment]
There are some doc changes I want to make to this so here's what I'm going to do.
1. create a new branch based off `main`
2. update this PR to PR into that branch
3. make changes
4. merge into `main`

That way we can fix these errors and get this in 👍

**Review by jsjoeio — APPROVED** [review]

**Comment by im-coder-lg** [comment]
Well, clone the repo locally/use Gitpod, and install the dependencies. After that, run `yarn fmt`. Hope I'm right, @jsjoeio is there more to the FMT?

**Comment by jsjoeio** [comment]
> there more to the FMT?

Just need to clean up a few things so yes, but I think it's all good now!

**Pull request merged**
